cluster/sync.sh

```diff
@@ -47,10 +47,23 @@ function isHandlerOk {
     [ "$desiredNumberScheduled" == "$numberAvailable" ]
 }
 
-function deploy_operator() {
-    # Cleanup previous deployment, if there is any
-    make cluster-clean
+function isExternal {
+    [[ "${KUBEVIRT_PROVIDER}" == external ]]
+}
 
+function isOpenshift {
+    $kubectl get co openshift-apiserver
+}
+
+function push() {
```

Comment:
Please add howto use this in https://github.com/nmstate/kubernetes-nmstate/blob/master/CONTRIBUTING.md